components/camel-asterisk/pom.xml

```diff
@@ -0,0 +1,104 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.apache.camel</groupId>
+    <artifactId>components</artifactId>
+    <version>2.18.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>camel-asterisk</artifactId>
+  <packaging>jar</packaging>
+  <name>Camel :: Asterisk</name>
+  <description>Camel Asterisk to interact with an Asterisk PBX Server.</description>
+
+  <properties>
+    <camel.osgi.import.before.defaults>
+      org.asteriskjava.*;version="[1.0,4)"
+    </camel.osgi.import.before.defaults>
+    <camel.osgi.export.pkg>
+      org.apache.camel.component.asterisk.*;${camel.osgi.version},
+    </camel.osgi.export.pkg>
+    <camel.osgi.export.service>org.apache.camel.spi.ComponentResolver;component=asterisk</camel.osgi.export.service>
+  </properties>
```

Comment:
asteriskjava does not appear to be OSGi ready, maybe you can open a PR to servicemix-bundles to create an OSGi ready asteriskjava bundle